### ISSUE TITLE
Plan to build UI?

### ISSUE THREAD

**Issue by luozhiheng** [pevey/medusa-plugin-blog#3]
hi,I found it's only backend。I can't  find admin dashborad has POST/BLOG button

**Comment by pevey** [comment]
Yes, this was made prior to the availability of the admin UI.  It is backend only.

**Comment by Derogater** [comment]
Hey, How do i use it?
